cartridges/openshift-origin-cartridge-python/usr/versions/3.3-scl/lib/update-configuration

```diff
@@ -8,19 +8,19 @@ function update-configuration {
     # SCL installs of python 3.3
     sclpath=$(dirname $(scl enable python33 "which python"))
     env_dir="${OPENSHIFT_PYTHON_DIR}/env"
-    echo "$OPENSHIFT_PYTHON_DIR/virtenv/venv/bin:$OPENSHIFT_PYTHON_DIR/bin:${sclpath}" > $OPENSHIFT_PYTHON_DIR/env/OPENSHIFT_PYTHON_PATH_ELEMENT
+    echo "${OPENSHIFT_PYTHON_DIR}virtenv/bin:${OPENSHIFT_PYTHON_DIR}bin:${sclpath}" > ${env_dir}/OPENSHIFT_PYTHON_PATH_ELEMENT
```

Comment:
Why was `venv` removed from the path? I think it is necessary, the new path specified does not exist:
```
> ls /var/lib/openshift/55a3d7395a00080933000155/python/virtenv/bin
ls: cannot access /var/lib/openshift/55a3d7395a00080933000155/python/virtenv/bin: No such file or directory

> ls /var/lib/openshift/55a3d7395a00080933000155/python/virtenv/venv/bin
activate  pip  pip3  pip3.3  pydoc  python  python3  python3.3
```